src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -125,6 +125,8 @@ public string NotSupportedReason
         /// https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions#x-ms-visibility.
         /// </summary>
         public bool IsInternal => Operation.IsInternal();
+        
+        public bool IsHidden => Visibility.ToVisibility() == Connectors.Visibility.Internal;
```

Comment:
What's the difference between IsInternal and IsHidden?
(especially since Hidden checks the internal flag)

Reply:
Absolutely none :)
I've removed IsHidden and all "hidden" notions to use IsInternal instead
Good catch!